Assignment/solution_student/select_fun_part2/js/main.js

```diff
@@ -1,29 +1,55 @@
 // 1.	Slide the header up when it is clicked on.
 
+$('header').on('click', function() {
+	$('header').slideUp(400)
+});
 // 2.	Remove the purplebox class
+$('#box14').removeClass('purplebox');
 
 // 3.	Hide all of the div’s in #row4
+$('#row4 div').hide();
 
 // 4.	Add a paragraph that says “I have been added” to all boxes with class “add-para” (Hint: you'll need to look up the append() method)
+$('.add-para').append('I have been added');
```

Comment:
Great job with the `.append()` method, the only thing I would suggest is that you add a `<p>` around the text you're adding so that when it gets appended, it's not "naked", and it's wrapped in a paragraph, which it should be, so something like:
```
$('.add-para').append('<p>I have been added</p>');
```